.gitlab-ci.yml

```diff
@@ -5,14 +5,20 @@ stages:
   - single-step-instrumentation-tests
   - macrobenchmarks
 
-# TODO(BridgeAR) Activate the gitlab CI job
-#
-# check_config_inversion_local_file:
-#   rules:
-#     - when: on_success
-#   extends: .check_config_inversion_local_file
-#   variables:
-#     LOCAL_JSON_PATH: "packages/dd-trace/src/supported-configurations.json"
+check_config_inversion_local_file:
+  rules:
+    - when: on_success
+  extends: .check_config_inversion_local_file
+  variables:
+    LOCAL_JSON_PATH: "packages/dd-trace/src/supported-configurations.json"
+
+config-inversion-update-supported-range:
```

Comment:
I don't see that this ran 🤔

Reply:
It should not run in this PR. It runs on our release branches by checking for the related tags. If it matches a release tag, it runs this script.

`if: '$CI_COMMIT_TAG =~ /^v?[0-9]+\.[0-9]+\.[0-9]+$/'`

Reply:
Ah I misread that if statement as being the reverse 🤦